torchtitan/config_manager.py

```diff
@@ -159,6 +159,18 @@ def __init__(self):
         self.parser.add_argument(
             "--optimizer.lr", type=float, default=8e-4, help="Learning rate to use"
         )
+        self.parser.add_argument(
+            "--optimizer.beta1", type=float, default=0.9, help="beta1 rate to use"
+        )
+        self.parser.add_argument(
+            "--optimizer.beta2", type=float, default=0.95, help="beta2 rate to use"
+        )
+        self.parser.add_argument(
+            "--optimizer.weight_decay",
```

Comment:
is `weight_decay` common for other optimizers? I am a little worried that these configs are not generalizable to other types of optimizers. So we might want to think of how to expose the optimizer configs in a more general way, i.e. suppose later we add another optimizer and all of the configs here might not be valid for that optimizer.